internal/container/container.go

```diff
@@ -173,7 +173,7 @@ func runCommand(c string, args ...string) error {
 	cmd := exec.Command(c, args...)
 	cmd.Stderr = os.Stderr
 	cmd.Stdout = os.Stdout
-	slog.Info(strings.Repeat("-", 80))
+	slog.Info(strings.Repeat("=", 80))
```

Comment:
Not sure of the purpose of this line. I don't particularly object, mind you. (Have you deliberately left the one below using '-'?)

Reply:
When looking at logs, it's just easier for my eye to differentiate the sections when there's a clear "start" line rather than separators that can mean both 'start' and 'middle'.

Not super important, though.

Reply:
Right, that's fine - no problem :)